src/test/java/starter/ReportSender.java

```diff
@@ -0,0 +1,13 @@
+package starter;
+
+import io.cucumber.java.After;
+
+import static integrations.SlackIntegration.sendMessageToSlack;
+
+public class ReportSender {
+
+    @After
+    public static void sendSerenityReportToSlack() {
+        sendMessageToSlack("Serenity Report: file:///C:/Mentoring/project/Serenity_Framework/target/site/serenity/index.html");
```

Comment:
is it possible to send not the link but Failed test notifications at the end? e.g. Total Test Run -XXX, Failed Test - XXX: FailedTestName1, FailedTestName2, ...
besides link is hard coded and no possibility to get the correct one when run on Jenkins or other machines